package.json

```diff
@@ -128,6 +129,228 @@
                 "language": "csharp",
                 "path": "./snippets/csharp.json"
             }
+        ],
+        "debuggers": [
+            {
+                "type": "coreclr",
+                "label": ".NET Core",
+                "enableBreakpointsFor": { "languageIds": [ "csharp", "fsharp" ] },
```

Comment:
Note that the C# extension does not actually support editing of F# code. I'm not sure this is the right place for this.